src/configuration/services/kafka.md

```diff
@@ -7,6 +7,7 @@ See the [Kafka documentation](https://kafka.apache.org/documentation/) for more
 ## Supported versions
 
 * 2.1
+* 2.2
 
 ## Relationship
```

Comment:
The service.yaml example should have this most recent version in it as well, like you did here https://github.com/platformsh/platformsh-docs/pull/1142/files